staging_service/autodetect/GenerateMappings.py

```diff
@@ -28,223 +28,114 @@
-        "output_type": ["KBaseMetagenomes.AnnotatedMetagenomeAssembly"],
-    },
-    # Commented out because: Batch App
-    # {
-    #     "title": "GFF/FASTA Genome Set",
-    #     "app": "kb_uploadmethods/batch_import_genome_from_staging",
-    #     "output_type": ["KBaseSearch.GenomeSet"],
-    #     "hidden": True
-    # },
-]
-
-mapping[ZIP] = [
-    {
-        "id": decompress_id,
+    decompress_id: {
         "title": "Decompress/Unpack",
         "app": "kb_uploadmethods/unpack_staging_file",
         "output_type": [None],
-    }
-]
-mapping[CSV] = [
-    {
+    },
+    sample_set_id: {
         "id": sample_set_id,
         "title": "Samples",
         "app": "sample_uploader/import_samples",
         "output_type": ["KBaseSets.SampleSet"],
-    }
-]
-
-mapping[TSV] = [
-    {
-        "id": media_id,
+    },
+    media_id: {
         "title": "Media",
         "app": "kb_uploadmethods/import_tsv_excel_as_media_from_staging",
         "output_type": ["KBaseBiochem.Media"],
     },
-    # Commented out because: Not in scope and requires an object
-    # {
-    #     "title": "Attribute Mapping",
-    #     "app": "kb_uploadmethods/import_attribute_mapping_from_staging",
-    #     "output_type": ["KBaseExperiments.AttributeMapping"],
-    # },
-    {
-        "id": expression_matrix_id,
+    expression_matrix_id: {
         "title": "Expression Matrix",
         "app": "kb_uploadmethods/import_tsv_as_expression_matrix_from_staging",
         "output_type": ["KBaseFeatureValues.ExpressionMatrix"],
     },
-    {
-        "id": metabolic_annotations_id,
+    metabolic_annotations_id: {
         "title": "Metabolic Annotations",
         "app": "MergeMetabolicAnnotations/import_annotations",
         "output_type": ["KBaseGenomes.Genome"],
     },
-    {
-        "id": metabolic_annotations_bulk_id,
+    metabolic_annotations_bulk_id: {
         "title": "Bulk Metabolic Annotations",
         "app": "MergeMetabolicAnnotations/import_bulk_annotations",
         "output_type": ["KBaseGenomes.Genome"],
     },
-    {
-        "id": fba_model_id,
+    fba_model_id: {
         "title": "FBA Model",
         "app": "kb_uploadmethods/import_file_as_fba_model_from_staging",
         "output_type": ["KBaseFBA.FBAModel"],
     },
-    {
-        "id": phenotype_set_id,
+    phenotype_set_id: {
         "title": "Phenotype Set",
         "app": "kb_uploadmethods/import_tsv_as_phenotype_set_from_staging",
         "output_type": ["KBasePhenotypes.PhenotypeSet"],
     },
-]
-
-mapping[EXCEL] = [
-    {
-        "id": sample_set_id,
-        "title": "Samples",
-        "app": "sample_uploader/import_samples",
-        "output_type": ["KBaseSets.SampleSet"],
-    },
-    {
-        "id": media_id,
-        "title": "Media",
-        "app": "kb_uploadmethods/import_tsv_excel_as_media_from_staging",
-        "output_type": ["KBaseBiochem.Media"],
-    },
-    {
-        "id": attribute_mapping_id,
+    attribute_mapping_id: {
```

Comment:
Should this be commented out, since it's not in scope?

Reply:
I just deleted it. It's easy enough to get the old data from the file history in github if we want to reinstate it.

Reply:
Alright that's fine. 
I highly doubt we will ever go into history to reinstate it, we would probably just rewrite it. How would one even know to look in the history for this app ? Hmm Maybe if we searched github for "attribute_mapping" it might come up?

Reply:
Hm, missed this comment before I merged.

I was thinking that if anyone reinstated it it'd be you or me and we'd just remember. If not, it's only 4 lines that you just pull from the app specs so not a big problem IMO.